src/test/SubscribableRequestQueue.test.ts

```diff
@@ -29,20 +27,20 @@ function isRejected(promise: Promise<unknown>): Promise<boolean> {
 
 describe("SubscribableRequestQueue", () => {
   describe("addSubscriber", () => {
-    it("adds a subscriber", () => {
+    test("adds a subscriber", () => {
```

Comment:
I'm surprised you had to change all of the `it` declarations to `test`? I believe you can import `it` from `vitest`... but maybe this is nice so your tests aren't explicitly dependent on `vitest`.

Reply:
I was surprised that `it` gave me some kind of error and maybe I was too heavyhanded but the search and replace was easy and made things work. easy enough to change it back and try again.

Reply:
re-replaced!  things are working and the diff is smaller